scss/mixins/_gradients.scss

```diff
@@ -20,33 +17,31 @@
 // Creates two color stops, start and end, by specifying a color and position for each color stop.
 @mixin gradient-x($start-color: $gray-700, $end-color: $gray-800, $start-percent: 0%, $end-percent: 100%) {
   background-image: linear-gradient(to right, $start-color $start-percent, $end-color $end-percent);
-  background-repeat: repeat-x;
```

Comment:
I removed the background repeats from these mixins, since these gradients always stretch over the whole element, see https://www.w3schools.com/css/tryit.asp?filename=trycss3_gradient-linear.